app/views/api_users/applications/index.html.erb

```diff
@@ -24,6 +24,13 @@
    })
 %>
 
+<% if flash[:application_id] %>
+<%= render "govuk_publishing_components/components/success_alert", {
+    message: "Permissions updated",
+    description: message_for_success(flash[:application_id], @api_user),
+} %>
+<% end %>
+
```

Comment:
I think it would be more consistent to put this in a `content_for(:custom_alerts)` block so that this appears above the heading, etc.

See:
* https://github.com/alphagov/signon/blob/ca17a91ad436f0c059f13a683fedafc14b9652fa/app/views/layouts/admin_layout.html.erb#L36
* https://github.com/alphagov/signon/blob/ca17a91ad436f0c059f13a683fedafc14b9652fa/app/views/api_users/manage_tokens.html.erb#L28-L37

Reply:
We've addressed this for /account, /users and /api_users in the commit titled, "Display permission flash notice in custom_alerts container".